pages/linux/ctop.md

```diff
@@ -0,0 +1,8 @@
+# ctop
+
+> Instantly visualize container performance and health with real-time metrics on CPU, memory, and block IO usage.
+> More information: <https://github.com/bcicen/ctop>.
+
+- Show tree view by default:
+
+`ctop --tree`
```

Comment:
There appears to be no such flag.
```
$ ctop --tree
flag provided but not defined: -tree
Usage of ctop:
  -a	show active containers only
  -connector string
    	container connector to use (default "docker")
  -f string
    	filter containers
  -h	display this help dialog
  -i	invert default colors
  -r	reverse container sort order
  -s string
    	select container sort field
  -v	output version information and exit
```

and https://github.com/bcicen/ctop?tab=readme-ov-file#options

also could you give some more examples once this is addressed since there are more things which need documenting.